model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -51,10 +53,14 @@ data class AnalyzerConfiguration(
      * Package manager specific configurations. The key needs to match the name of the package manager class, e.g.
      * "NuGet" for the NuGet package manager.
      */
-    val packageManagers: Map<String, PackageManagerConfiguration>? = null,
+    @ConfigAlias("packageManagers")
+    private val packageManagersInternal: Map<String, PackageManagerConfiguration>? = null,
```

Comment:
Would it make sense to fail in case there are multiple configurations for the same package manager provided?
e.g. add a require block checking `packageManagersInternal` ?

Reply:
Good idea, done.